eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -317,8 +317,8 @@ jobs:
     # Using build artifacts to enable publishing the vertical manifests to a single artifact from different jobs
     - ${{ if ne(parameters.buildSourceOnly, true) }}:
       - output: buildArtifacts

```

Comment:
Is it necessary to use build artifacts vs. a pipeline artifacts here? I thought you had found some way of downloading all artifacts from the pipeline that matched some pattern.